apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/context/TracingContext.java

```diff
@@ -462,6 +463,17 @@ public boolean stopSpan(AbstractSpan span) {
         finish();
     }
 
+    @Override
+    public void prepareProfiling(String firstSpanOPName) {
```

Comment:
Why need this method? I think we just need to add a parameter in the TracingContext constructor.